doc/sai-ptf/L3_test_plan.md

```diff
@@ -0,0 +1,333 @@
+6. Generate Packets, ``SIP:192.168.0.1-192.168.0.10`` 
+8. Sent packets with ``DIP:192.168.60.1`` and change the ``L4_port`` number in different packets
+9. Verify Packets only get received on LAG1 and LAG2 and with ``SMAC: SWITCH_MAC_2``
+
+- test_lag_ecmp_lpm_add
+
+1. run steps 1-10 in test_ecmp_lpm
+2. add next hop for ip ``10.1.2.101`` to the next-hop group in test_ecmp_lpm
+3. Generate Packets, with different source ip ``SIP:192.168.0.1-192.168.0.10`` 
+4. Change other elements in the packets, including ``DIP:192.168.60.1`` ``L4_port``
+5. Verify Packets can be received on LAG1 LAG2 and LAG3, with ``SMAC: SWITCH_MAC_2``
+
+- test_lag_ecmp_lpm_disable
+
+1. run steps 1-8 in test_ecmp_lpm_add
+2. Disable LAG3 members
+3. Generate Packets, with different source ip as ``SIP:192.168.0.1-192.168.0.10`` 
+4. Change other elements in the packets, including ``DIP:192.168.60.1`` ``L4_port``
+5. Verify Packets no packet lost and only can be received on LAG1 and LAG2, with ``SMAC: SWITCH_MAC_2``
+
+- test_lag_ecmp_lpm_remove
+
+1. run steps 1-8 in test_ecmp_lpm_add
+2. remove the next hop from next hop group in test_ecmp_lpm: nexthop with IP ``DIP:10.1.3.101`` on LAG3 
+3. Generate Packets, with different source ip as ``SIP:192.168.0.1-192.168.0.10`` 
+4. Change other elements in the packets, including ``DIP:192.168.60.1`` ``L4_port``
+5. Verify Packets only can be received on LAG1 and LAG2, with ``SMAC: SWITCH_MAC_2``
+
+- test_next_hop_group_and_ecmp_entry_api
+
+1. Get those attributes [number_of_ecmp_groups, ecmp_members, available_next_hop_group_entry, available_next_hop_group_member_entry]
+2. remove ecmp member and next group member
+3. Get attributes again and check the value
+
+## Test Group4: Route interface
+### Case1: test_ingress_disable
+### Case2: test_ingress_mac_update
+### Case3: test_ingress_mtu
+### Case4: test_ingress_disable_v6
+### Case5: test_ingress_mac_update_v6
+### Case6: test_ingress_mtu_v6
+### Case7: test_sub_port
+
+### Testing Objective <!-- omit in toc --> 
+- test_ingress_disable:   Verify turn off the admin state for RIF (v4 or v6)
+- test_ingress_mac_update:Verify teh packet will be dropped if the packet dest mac not match to the mac in route interface 
+- test_ingress_mtu:       Verify teh packet will be dropped if the packet length exceed the mtu value
+
+### Test steps <!-- omit in toc --> 
+
+- test_ingress_disable
+
+1. Set the admin state to False on route interface binds to Port5
+2. Generate Packets, with ``SIP:192.168.0.1`` ``DIP:10.1.1.101`` ``DMAC:SWITCH_MAC``
+3. Send packet on Port5
+4. Verify no packet was received on one of the LAG1's member
+
+- test_ingress_mac_update
+
+1. Generate Packets, with ``SIP:192.168.0.1`` ``DIP:10.1.1.101`` ``DMAC:SWITCH_MAC``
+2. Send packet on Port5
+3. Verify packet received on one of the LAG1's member
+4. Set RIF mac to ``MacX``, the RIF related to Port5
+5. Send packet on Port5 
+6. Verify no packet was received on any LAG1 member
+
+- test_ingress_mtu
```

Comment:
not sure what's the MTU on T0, I heard we can support jumbo frame, should we remove this case or lower the priority?

Reply:
this is also for the attribute, can be lower

Reply:
I see the MTU setting is 9100 for T0